marklogic-data-hub/src/test/java/com/marklogic/hub/flow/RedactionTest.java

```diff
@@ -0,0 +1,85 @@
+package com.marklogic.hub.flow;
```

Comment:
Let's put this in hub.security and call it RedactionUserTest for consistency with the other role test classes

Reply:
done